k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -235,6 +235,7 @@
     private RecipientPresenter recipientPresenter;
     private MessageBuilder currentMessageBuilder;
     private boolean mFinishAfterDraftSaved;
+    private boolean firstTimeEmptyObject = true;
```

Comment:
The Activity is recreated on configuration changes and the value of the field is lost. So, e.g. pressing 'send' once will display the error message. If you rotate the device and press 'send' again, the message will show another time.

Use `onSaveInstanceState()` and `onRetainInstanceState()` to save and restore the value of the field.